core/components/molecules/chipInput/__stories__/ChipsWrapped.story.jsx

```diff
@@ -9,7 +9,7 @@ export const chipsWrappedToNextLine = () => {
   ]);
 
   return (
-    <div style={{ width: '320px' }}>
+    <div className="Col--4">
```

Comment:
Please use `<Column size={4}>` or width-based classes, Column should not be used without Row as a parent, and classes of columns should not be used in react as this is a component class and not a utility.